Makefile

```diff
@@ -507,6 +507,7 @@ $(1): $(PREFIX)/$(3)/installed/$(1)
 $(PREFIX)/$(3)/installed/$(1): $(PKG_MAKEFILES) \
                           $(PKG_PATCHES) \
                           $(PKG_TESTFILES) \
+                          $($(PKG)_FILE_DEPS) \
```

Comment:
Yes, generally references to `$(1)` and `$(3)` can be replaced with `$(PKG)` and `$(TARGET)`. I think that should be a separate PR though.